e2e-tests/cypress/integration/appeal-submission-save-and-continue-navigation-appeal-site/appeal-submission-save-and-continue-navigation-appeal-site.js

```diff
@@ -18,7 +18,7 @@ Given('the "Site ownership" is not wholly owned', () => {
   cy.answerDoesNotOwnTheWholeAppeal();
   cy.clickSaveAndContinue();
 
-  cy.userIsNavigatedToPage('/appellant-submission/site-ownership-certb');
+  cy.userIsNavigatedToPage('/appeal-householder-decision/site-ownership-certb');
```

Comment:
Please can this file and dir be renamed to reflect `appeal-householder-decision` ? thanks